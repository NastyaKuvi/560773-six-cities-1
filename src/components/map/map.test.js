import React from "react";
import renderer from "react-test-renderer";
import CitiesMap from "./map";

const div = global.document.createElement(`div`);
div.setAttribute(`id`, `map`);
global.document.body.appendChild(div);

const mock = {
  city: [52.38333, 4.9],
  coords: [
    [52.3909553943508, 4.85309666406198],
    [52.369553943508, 4.85309666406198],
    [52.3909553943508, 4.929309666406198],
    [52.3809553943508, 4.939309666406198]
  ]
};

it(`map component correctly renders`, () => {
  let {city, coords} = mock;
  const comp = renderer.create(
      <CitiesMap
        coords = {coords}
        city = {city}
      />).toJSON();

  expect(comp).toMatchSnapshot();
});
