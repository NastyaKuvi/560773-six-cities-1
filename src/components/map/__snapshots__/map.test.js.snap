// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`map component correctly renders 1`] = `
<div
  id="map"
  style={
    Object {
      "height": "100%",
    }
  }
/>
`;
